scripts/lua/modules/host_pools_nedge.lua

```diff
@@ -448,22 +448,49 @@ function host_pools_nedge.printQuotas(pool_id, host, page_params)
 end
 
 function host_pools_nedge.resetPoolsQuotas(pool_filter)
```

Comment:
This method has a pool_filter parameter which is not handled, please handle or remove it if unused by the caller.